assets/javascripts/discourse/components/modal/post-event-invitees/index.gjs

```diff
@@ -12,29 +12,29 @@ import concatClass from "discourse/helpers/concat-class";
 import i18n from "discourse-common/helpers/i18n";
```

Comment:
I created a new "Invitees" model which can also old the suggested users. Not sure yet about this design but this is ok for now